app/components/order-card.js

```diff
@@ -0,0 +1,10 @@
+import Component from '@ember/component';
+import { computed } from '@ember/object';
+import { isEqual } from '@ember/utils';
+
+export default Component.extend({
+  isFreeOrder: computed('order', function() {
+    const amount = this.get('order.amount');
+    return amount === null || isEqual(amount, '0');
```

Comment:
isn't the amount numeric?

Reply:
No, right now it is just a string. 
https://github.com/fossasia/open-event-frontend/blob/96581616a3b2466113a8ed430ae8d9f00d98a271/app/models/order.js#L9

Want me to change it ?

Reply:
If on the server side it's not a string, then sure please change it.